lamindb/curators/__init__.py

```diff
@@ -1875,7 +1882,7 @@ def categoricals(self) -> dict[str, dict[str, FieldAttr]]:
         return self._categoricals
 
     @property
-    def non_validated(self) -> dict[str, dict[str, list[str]]]:
+    def non_validated(self) -> dict[str, dict[str, list[str]]]:  # type: ignore
```

Comment:
Rather than adding type ignore, we should think about defining the schema for `non_validated`. It's a bit of a complicated, vague thing right now.